src/api/data-types.md

```diff
@@ -79,3 +79,19 @@ by locality.
 [ISO 8601]: https://en.wikipedia.org/wiki/ISO_8601
 [ISO 4217]: https://en.wikipedia.org/wiki/ISO_4217
 [ISO 3166]: https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2
+
+## BankAccountApprovalTypeSummary
```

Comment:
Don't use camel case. This is not a generic data type; it should be defined as part of the bank account model.